pkg/kube/persistantvolumeclaim/pvc.go

```diff
@@ -0,0 +1,33 @@
+package persistantvolumeclaim
```

Comment:
The package is called `persistAntvolumeclaim` instead of `persistEntvolumeclaim`, which is the term used in k8s. Is this a typo?

Reply:
almost definitely, nice catch!